app/jobs/send_email_month_job.rb

```diff
@@ -0,0 +1,8 @@
+class SendEmailMonthJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    @bookings = Booking.where("created_at BETWEEN ? AND ?", Date.today.beginning_of_month, Date.today.end_of_month)
```

Comment:
`Booking.where("created_at BETWEEN ? AND ?", Date.today.beginning_of_month, Date.today.end_of_month)`
viết scope trong model tương ứng đi thanh niên nhé -_-